document/core/syntax/instructions.rst

```diff
@@ -541,6 +541,15 @@ The instructions |I31NEW| and :math:`\I31GET\K{\_}\sx` convert between type |I31
 
 The instructions |EXTERNINTERNALIZE| and |EXTERNEXTERNALIZE| allow lossless conversion between references represented as type :math:`(\REF~\NULL~\EXTERN)`| and as :math:`(\REF~\NULL~\ANY)`.
 
+The following auxiliary function produces an instruction that accesses an array slot for a given :ref:`storage type <syntax-storagetype>`.
+
+.. _aux-getfield:
+
+.. math::
+   \begin{array}{lll}
+   \getfield(\valtype) &=& \ARRAYGET \\
+   \getfield(\packedtype) &=& \ARRAYGETU \\
+   \end{array}
```

Comment:
I'd move this next to the rules (as a "where:" sort of thing), since it is local helper for that specific rule only.